src/Raven.Server/Documents/Handlers/Admin/AdminConfigurationHandler.cs

```diff
@@ -67,6 +69,9 @@ public async Task GetSettings()
                 var entry = new ConfigurationEntryDatabaseValue(Database.Configuration, databaseRecord, configurationEntryMetadata, status);
                 settingsResult.Settings.Add(entry);
             }
+            
+            if (LoggingSource.AuditLog.IsInfoEnabled)
+                LogAuditFor(Database.Name, "PUT", "Database configuration changed");
```

Comment:
1. You added it to GET handler. It should be on PUT
2. Let's modify the parameters a bit (please note that under the covers they are concatenated) to 
```
LogAuditFor(Database.Name, "CHANGE", "Database configuration");
```

Reply:
Shouldn't we also include what the configuration changes _were_ ?

Reply:
We can read current settings from the db record, compare the values and then include the names of modified settings (not the actual values I think). 

@gregolsky do you think we need such granularity in audit log?

Reply:
Let's log the key of the settings. The value is not that interesting and can be retrieved from backup if needed, right?

Reply:
Added keys of modified settings to message